File: Check for prime/checkPrime.py
import math


# Cheking Whether the number is prime or not
def checkPrime(Number):
    for i in range(2, int(math.sqrt(Number)) + 1):
        if Number % i == 0:
            return "No"
    return "Yes"


if __name__ == "__main__":
    array = []

    # Static Input
    Number = 7

    print(checkPrime(Number))

File: 02_Check for prime/Ideal Solutions/checkPrime.java
import java.io.*;
import java.lang.*;
import java.util.*;


class Main{
	//This function checks wether a particular number is prime or not.
	public static boolean checkPrime(int n){
		for (int i=2;i<=Math.sqrt(n);i++){
			if (n % i == 0) return false;
		}
		return true;
	}
	//Main Function.
	public static void main(String[] args){
		int n = 10;
		if (checkPrime(n)) System.out.println("Yes");
		else{
			System.out.println("No");
		}
	}
}
